packages/common/src/components/Footer.jsx

```diff
@@ -7,23 +7,51 @@
  */
 
 import React from 'react'
+import { FooterLink } from '@kernel/common'
 
+const env = process.env.REACT_APP_DEPLOY_TARGET || 'PROD'
+const prefix = env === 'STAGING' ? 'staging.' : ''
 const defaults = {
   backgroundColor: 'bg-gray-900',
-  textColor: 'text-white'
+  textColor: 'text-white',
+  footerLinks: [
+    { href: 'https://' + prefix + 'explore.kernel.community', title: 'explorer' },
+    { href: 'https://' + prefix + 'unprofile.kernel.community', title: 'unprofile' },
+    { href: 'https://' + prefix + 'adventures.kernel.community', title: 'adventures' },
+    { href: 'https://' + prefix + 'wallet.kernel.community', title: 'portal' },
+  ]
 }
 
-export default function Footer (props) {
+export default function Footer(props) {
   const { children } = props
+  /* TODO: address when Footer has a single child which is not a link */
+  const links = children ?
+    children.length > 1 ?
+      children
+        .filter(link => link.props !== undefined)
+        .map((link) => ({ href: link.props.href, title: link.props.children }))
+      : [{ href: children.props.href, title: children.props.children }]
+    : defaults.footerLinks
```

Comment:
Hmm, I think this might be too constraining on the caller of this component, requiring them to understand all the internal logic here-- and also might result in some surprising behavior. Like if you use this footer and put your own styled links, they will get re-styled, which would probably be surprising. Also, we should still allow the caller to inject children that aren't links, like if they just want text.

Simpler alternative would be to offer the following options:
+ If `children` are present, just render only the children, instead of the default `ul` and the "built at KERNEL". This way the caller gets full control of content, but still with spacing/colors consistent with the common footer.
+ If no `children`, allow an optional prop like `links` where the caller can pass the config directly (ie., just pass the array of objects that each has `href` and `title`.. see the pattern used for `Navbar`) instead of pulling those attributes out of the children.
+ If no `children` and no `links` prop is present, just render the full KERNEL default.

Reply:
You're absolutely right. Completely refactored this, which also made it much cleaner and shorter 😄 
Capturing the logic in the PR itself.